### PR TITLE
add multi loglevels and number of logs displayed

### PR DESCRIPTION
**Pull request by liquidhelium**

### CODE REVIEW

**Comment by RegenJacob** [comment]
Thank you for the pull request! 
I will test this later

File: src/lib.rs [hunk 1]
```diff
@@ -37,16 +45,17 @@ static LOGGER_UI: once_cell::sync::Lazy<Mutex<LoggerUi>> =
...
-    loglevel: log::Level,
+    loglevels: [bool;log::Level::Trace as usize],
     search_term: String,
     copy_text: String,
     max_log_length: usize,
 }
 
 impl Default for LoggerUi {
     fn default() -> Self {
+        // Error, Warn and Info
```
Comment (RegenJacob) [inline src/lib.rs]:
```suggestion
```

File: src/lib.rs [hunk 2]
```diff
@@ -93,6 +103,9 @@ impl LoggerUi {
         });
         ui.separator();
 
+        // a counter
```
Comment (RegenJacob) [inline src/lib.rs]:
```suggestion
```

File: src/lib.rs [hunk 3]
```diff
@@ -66,14 +75,15 @@ impl LoggerUi {
...
-                });
+            ui.label("Log Level");
+            // I didn't find elegant ways to iterate an Enum :(
+            for level in LEVELS
+            {
+                if ui.selectable_label( self.loglevels[level as usize - 1], level.as_str()).clicked()
+                {
+                    self.loglevels[level as usize - 1] = !self.loglevels[level as usize - 1];
+                }
+            }
```
Comment (RegenJacob) [inline src/lib.rs]:
```suggestion
            ui.menu_button("Log Levels", |ui| {
              for level in LEVELS
              {
                  if ui.selectable_label( self.loglevels[level as usize - 1], level.as_str()).clicked()
                  {
                      self.loglevels[level as usize - 1] = !self.loglevels[level as usize - 1];
                  }
              }
            }
```

Please replace the label with a menu_button containing the selectable Levels

**Review by RegenJacob — CHANGES_REQUESTED** [review]
Nice work! just some small tweaks would be nice e.g. some comments are unnecessary in my opinion and moving the log level selectors inside a menu_button

**Comment by liquidhelium** [comment]
Thank you! I'll do this later.

**Comment by liquidhelium** [comment]
Fine, everything is ok now

**Pull request merged**